.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ env:
 
 jobs:
   tests:
+    name: CI Test Suite
```

Comment:
fwiw since this is used directly in the UI the horizontal space matters.  I'd have made it `tests` to keep the same.

Reply:
> fwiw since this is used directly in the UI the horizontal space matters. I'd have made it `tests` to keep the same.

Agree, I just took something else as it overlaps with `test / Tests` and GH UI is confusing around these. I'll make an issue and revise on another pass.

Reply:
ah yeah.  that branch protection UI is pretty rubbish too :(